tests/helpers/recompute_zones_helpers.test.ts

```diff
@@ -0,0 +1,885 @@
+import { modifyProfiles, recomputeZones, toUnboundedZone, zoneToXc } from "../../src/helpers";
+
+const recomputeZonesFromXC = function (xcs: string[], xcsToRemove: string[]): string[] {
+  return recomputeZones(xcs.map(toUnboundedZone), xcsToRemove.map(toUnboundedZone)).map(zoneToXc);
+};
+
+// Only to test result after modifyProfiles
+function computeProfiles(xcs: string[], xcsToRemove: string[]): Map<number, number[]> {
+  const zones = xcs.map(toUnboundedZone);
+  const zonesToRemove = xcsToRemove.map(toUnboundedZone);
+
+  const profilesStartingPosition: number[] = [0];
+  const profiles = new Map<number, number[]>([[0, []]]);
+
+  modifyProfiles(profilesStartingPosition, profiles, zones, false);
+  modifyProfiles(profilesStartingPosition, profiles, zonesToRemove, true);
+  return profiles;
+}
+
+describe("modifyProfiles", () => {
+  test("test modifyProfiles", () => {
```

Comment:
I'd add a comment with either an explaination on what profile are, or a comment that says where we can find the explaination of what profiles are